library/Makefile

```diff
@@ -138,7 +137,7 @@ ifneq ($(APPLE_BUILD),0)
 endif
 endif
 
-libmbedx509.$(SOEXT_X509): $(OBJS_X509) $(CRYPTO)libmbedcrypto.so
+libmbedx509.$(SOEXT_X509): $(OBJS_X509) libmbedcrypto.so
```

Comment:
Should this be `libmbedcrypto.$(DLEXT)` rather than `￼libmbedcrypto.so` (see below regarding `libmbedcrypto.$(DLEXT):`)?